### PR TITLE
[SUBMARINE-192] Submarine workbench server daemon shell script

### PR DESCRIPTION
**Pull request by liuxunorg**
### What is this PR for?
Increase the Submarine workbench server daemon script,
Support to start, stop, restart the submarine workbench server through `submarine-daemon.sh`

```
./bin/workbench-daemon.sh [start|stop|restart]
```

### What type of PR is it?
[Feature]

### What is the Jira issue?
* https://issues.apache.org/jira/browse/SUBMARINE-192

### How should this be tested?
* [CI Pass](https://travis-ci.org/liuxunorg/hadoop-submarine/builds/590699956)

### Screenshots (if appropriate)

workbench-daemon-sh

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? Yes

### CODE REVIEW

File: bin/common.sh [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+  export SUBMARINE_LOG_DIR="${SUBMARINE_HOME}/logs"
+fi
+
+if [[ -f "${SUBMARINE_CONF_DIR}/submarine-env.sh" ]]; then
+  . "${SUBMARINE_CONF_DIR}/submarine-env.sh"
+fi
+
+WORKBENCH_CLASSPATH+=":${SUBMARINE_CONF_DIR}"
+
+function addEachJarInDir(){
```
Comment (jiwq) [inline bin/common.sh]: Refer the Google Style ([Function Names](https://google.github.io/styleguide/shell.xml?showone=Function_Names#Function_Names)), it's' not acceptable.

File: bin/workbench-daemon.sh [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+    ;;
+  reload)
+    stop
+    start
+    ;;
+  restart)
+    echo "${WORKBENCH_NAME} is restarting" >> "${WORKBENCH_LOGFILE}"
+    stop
+    start
+    ;;
```
Comment (jiwq) [inline bin/workbench-daemon.sh]: What's the difference between `reload` and `start` command?

File: bin/workbench-daemon.sh [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+
+function initialize_default_directories() {
+  if [[ ! -d "${SUBMARINE_LOG_DIR}" ]]; then
+    echo "Log dir doesn't exist, create ${SUBMARINE_LOG_DIR}"
+    $(mkdir -p "${SUBMARINE_LOG_DIR}")
+  fi
+}
+
+function foundWorkbenchServerPid() {
+  process=WorkbenchServer;
```
Comment (jiwq) [inline bin/workbench-daemon.sh]:
```suggestion
  process='WorkbenchServer';
```

File: bin/workbench-daemon.sh [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+  RUNNING_PIDS=$(ps x | grep $process | grep -v grep | awk '{print $1}');
```
Comment (jiwq) [inline bin/workbench-daemon.sh]:
```suggestion
  RUNNING_PIDS=$(ps x | grep ${process} | grep -v grep | awk '{print $1}');
```

**Comment by liuxunorg** [comment]
@jiwq , thank you advise.

File: bin/workbench-daemon.sh (hunk 2 above)
Comment (liuxunorg) [reply bin/workbench-daemon.sh]: No different, So i delete `reload` commond.

File: bin/workbench-daemon.sh (hunk 4 above)
Comment (liuxunorg) [reply bin/workbench-daemon.sh]: Done.

File: bin/workbench-daemon.sh (hunk 3 above)
Comment (liuxunorg) [reply bin/workbench-daemon.sh]: Done.

File: bin/common.sh (hunk 1 above)
Comment (liuxunorg) [reply bin/common.sh]: Done.

**Pull request closed without merging**